### PR TITLE
matchColumns() cleanup

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Address Issue 43009: ETL fails when aliasing built-in columns
The primary way we match import columns is by name, however there is code to also match propertyuri-propertyuri (todo verify this is still used?).  The problem is that this match is treated as _higher_ priority than a name match, and is somewhat susceptible to generating duplicate/ambiguous matches (see issue).

The proposed fix is to treat a propertyuri-propertyuri match as low priority.

#### Related Pull Requests
https://github.com/LabKey/platform/pull/2222
https://github.com/LabKey/dataintegration/pull/105
https://github.com/LabKey/platform/pull/2263

### CODE REVIEW

File: api/schemas/tableInfo.xsd [hunk 1]
```diff
@@ -996,14 +996,23 @@
                     These properties define how the column data is imported, and the target ontology.
                 </xs:documentation>
             </xs:annotation>
-            <xs:element name="labelColumn" type="xs:string" minOccurs="0">
+            <xs:element name="subtree" type="xs:string" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        This is a slash ('/') separated path of concepts codes.  The values in this column are expected
```
Comment (labkey-adam) [inline api/schemas/tableInfo.xsd:1002]:
```suggestion
                        This is a slash ('/') separated path of concept codes.  The values in this column are expected
```

File: api/schemas/expTypes.xsd [hunk 2]
```diff
@@ -589,6 +589,15 @@
 					These properties define how the column data is imported, and the target ontology.
 				</xs:documentation>
 			</xs:annotation>
+			<xs:element name="subtree" type="xs:string" minOccurs="0" maxOccurs="1">
+				<xs:annotation>
+					<xs:documentation>
+						This is a slash ('/') separated path of concepts codes.  The values in this column are expected
```
Comment (labkey-adam) [inline api/schemas/expTypes.xsd:595]:
```suggestion
						This is a slash ('/') separated path of concept codes.  The values in this column are expected

```

**Review by labkey-adam — APPROVED** [review]

File: api/schemas/expTypes.xsd (hunk 2 above)
Comment (labkey-matthewb) [reply api/schemas/expTypes.xsd:595]: Ugh.  Must have had an unpushed commit???  I dunno, this was only supposed to have changes for DataIteratorUtil.java.

Will fix.

Comment (labkey-adam) [reply api/schemas/expTypes.xsd:595]: I guess I should have asked the question...

**Comment by labkey-matthewb** [comment]
created clean version

**Pull request closed without merging**